CONTRIBUTING.md

```diff
@@ -32,3 +36,5 @@ Changes for the release are to be approved by 2 TES specification leads.
 ### post TES v1.0.0 Voting Procedure
 The post v1.0.0 voting group include stakeholders, such as server and client implementors.
 The membership of this group will be established as part of the v1.0.0 release.
```

Comment:
Also, I think we can remove the pre-TES v1.0.0 stuff, because we're already passed that. And then we can remove the explicit "post" as well, and can just name the section "Voting Procedure".

Reply:
Finally, we might want to move the entire section to `GOVERNANCE.md` and only keep the reference here (because, as it is, this very important info is now missing from `GOVERNANCE.md`).

Reply:
Fix this too